docs/features/ui-language.md

```diff
@@ -121,7 +121,8 @@ If you build CKEditor 5 from scratch or integrate it directly into your applicat
 	```js
 	const { CKEditorTranslationsPlugin } = require( '@ckeditor/ckeditor5-dev-translations' );
 
-	// Define webpack plugins...
+	module.exports = {
+		// Define webpack plugins...
```

Comment:
Could we move those three dots to the new line? :)

Reply:
Sure thing :)